testsuite/gateways/wasm/__init__.py

```diff
@@ -96,11 +96,10 @@ def remove_policy(self, name: str):
     # pylint: disable=unused-argument, too-many-arguments
     def _create_api_client(self, application, endpoint, verify, cert=None, disable_retry_status_list=None):
         ext = self.extensions[application.service["id"]]
-        ext.synchronise_mapping_rules()
-        ext.synchronise_credentials()
+        if ext.synchronise_credentials():
+            ext.httpbin.deployment(f"dc/{ext.httpbin_name}").rollout()
+            ext.httpbin.deployment(f"dc/{ext.httpbin_name}").wait_for()
```

Comment:
I believe rollout also waits for the resource

Reply:
If so, I removed the wait